src/components/Box/Box.tsx

```diff
@@ -0,0 +1,16 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import React from 'react';
+
+interface DataProps extends React.HTMLAttributes<HTMLDivElement> {
+    'data-test-id'?: string;
+}
+
+export const Box = ({ children, ...props }: DataProps) => {
```

Comment:
Here at your own criteria, if you want to simplify your ternary, you can also specify data-test-id in the args.
```
export const Box = ({ children, 'data-test-id': dataTestId, ...props }: BoxProps) => {
    return (
        <div data-test-id={dataTestId ? `fondue-${dataTestId}` : 'fondue-box'} {...props}>
            {children}
        </div>
    );
};
```

Reply:
I would simplify it even more, like

```
export const Box = ({ children, 'data-test-id': dataTestId = 'fondue-box', ...props }: BoxProps) => {
    return (
        <div data-test-id={dataTestId} {...props}>
            {children}
        </div>
    );
};
```